content/id/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -0,0 +1,302 @@
+    snowflake    2/2     2            2           2m
+    ```
+    ```shell
+    kubectl get pods -l app=snowflake -n=development
+    ```
+    ```
+    NAME                         READY     STATUS    RESTARTS   AGE
+    snowflake-3968820950-9dgr8   1/1       Running   0          2m
+    snowflake-3968820950-vgc4n   1/1       Running   0          2m
+    ```
+
+    Dan ini merupakan sesuatu yang bagus, dimana pengembang bisa melakukan hal yang ingin mereka lakukan tanpa harus khawatir hal itu akan mempengaruhi konten pada namespace `production`.
+    
+    Mari kita pindah ke Namespace `production` dan menujukkan bagaimana sumber daya di satu Namespace disembunyikan dari yang lain
+
+    Namespace `production` seharusnya kosong, dan perintah berikut ini seharusnya tidak menghasilkan apapun.
+
+    ```shell
+    kubectl get deployment -n=production
+    kubectl get pods -n=production
+    ```
+
+    `Production` Namespace ingin menjalankan `cattle`, mari kita buat beberapa Pod `cattle`.
+
+    ```shell
+    kubectl create deployment cattle --image=k8s.gcr.io/serve_hostname -n=production
+    kubectl scale deployment cattle --replicas=5 -n=production
+
+    kubectl get deployment -n=production
+    ```
+    ```
+    NAME         READY   UP-TO-DATE   AVAILABLE   AGE
+    cattle       5/5     5            5           10s
+    ```
+
+    ```shell
+    kubectl get pods -l app=cattle -n=production
+    ```
+    ```
+    NAME                      READY     STATUS    RESTARTS   AGE
+    cattle-2263376956-41xy6   1/1       Running   0          34s
+    cattle-2263376956-kw466   1/1       Running   0          34s
+    cattle-2263376956-n4v97   1/1       Running   0          34s
+    cattle-2263376956-p5p3i   1/1       Running   0          34s
+    cattle-2263376956-sxpth   1/1       Running   0          34s
+    ```
+
+Sampai sini, seharusnya sudah jelas bahwa sumber daya yang dibuat pengguna pada sebuah Namespace disembunyikan dari Namespace lainnya.
+
+Seiring dengan evolusi dukungan kebijakan di Kubernetes, kami akan memperluas skenario ini untuk menunjukkan bagaimana kamu bisa menyediakan aturan otorisasi yang berbeda untuk tiap Namespace.
+
+
+<!-- discussion -->
+
+## Memahami motivasi penggunaan Namespace
+
+Sebuah klaster tunggal umumnya bisa memenuhi kebutuhan pengguna yang berbeda atau kelompok pengguna (itulah sebabnya disebut 'komunitas pengguna').
+
+Namespace Kubernetes membantu proyek-proyek, tim-tim dan pelanggan yang berbeda untuk berbagi klaster Kubernetes.
+
+Ini dilakukan dengan menyediakan hal berikut:
+
+1. Cakupan untuk [Names](/id/docs/concepts/overview/working-with-objects/names/).
+2. Sebuah mekanisme untuk memasang otorisasi dan kebijakan untuk bagian dari klaster.
+
+Penggunaan Namespace berbeda merupakan hal opsional.
+
+Tiap komunitas pengguna ingin bisa bekerja secara terisolasi dari komunitas lainnya.
+
+Tiap komunitas pengguna memiliki hal berikut sendiri:
+
+1. sumber daya (Pod, Service, _controller_ replikasi, dll.)
```

Comment:
```suggestion
1. sumber daya (Pod, Service, ReplicationController, dll.)
```

(aside: it would be better if the English original used Deployment and not ReplicationController as its example).